docs/guidance/map.md

```diff
@@ -120,11 +118,10 @@ Some data elements might not match any field or code in OCDS. To cover such case
 
 **Action:** If you are stuck on a particular concept and are concerned about how it is modelled in OCDS, search the issues in our [Github tracker](https://github.com/open-contracting/standard/issues) to see what others in the community are saying about the topic. If you do not see your issue, create a new one!
 
-```{eval-rst}
-.. toctree::
-   :hidden:
+```{toctree}
+:hidden:
 
-   map/extensions
+map/extensions.md
```

Comment:
Why does this line need `.md` but other toctrees don't?

Reply:
To be honest I'm not sure, in this case it was throwing a warning and the only reason I could come up with was that there was both a file called map and a folder called map